app/controllers/questions/consent_controller.rb

```diff
@@ -9,6 +9,7 @@ def form_params
     end
 
     def after_update_success
+      ConsentPdf.new(current_intake).create_consent_document
```

Comment:
If we create the consent document here, we will not be creating a consent document for drop off clients. If we move this to a hook on the model that runs when a user consents, that will cover it getting generated in both workflows.

Reply:
UTA: Learned from product that we don't need to create any intake or consent documents for drop off clients because they should be signed on site and uploaded.

Reply:
@bytheway875 oh great! This was moved to the after_save method in the model anyways but I think it still wouldn't create a consent document if the primary_consented_to_service_at isn't filled for the drop off client